frontend/kesaseteli/employer/src/components/application/summary/ApplicationSummary.tsx

```diff
@@ -0,0 +1,76 @@
+import EmploymentSummary from 'kesaseteli/employer/components/application/summary/EmploymentSummary';
+import useApplicationApi from 'kesaseteli/employer/hooks/application/useApplicationApi';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import FormSection from 'shared/components/forms/section/FormSection';
+import { $GridCell, $Hr } from 'shared/components/forms/section/FormSection.sc';
+import FormSectionHeading from 'shared/components/forms/section/FormSectionHeading';
+import PageLoadingSpinner from 'shared/components/pages/PageLoadingSpinner';
+
+const ApplicationSummary: React.FC = () => {
+  const { t } = useTranslation();
+  const { application, isLoading } = useApplicationApi();
+  if (!application || isLoading) {
```

Comment:
Will be a case where `isLoading` is `false` but there is no `application`? If so should the application show some message or something more informative rather than a loading spinner?